app/models/project.rb

```diff
@@ -31,6 +31,7 @@ class Project < ActiveRecord::Base
   validates :name, presence: true, uniqueness: { scope: :organization_id, case_sensitive: false }, length: { maximum: 50, allow_blank: true }, format: { with: /\A[a-z0-9]+(?:\-[a-z0-9]+)*\Z/i }
   validates :display_name, length: { maximum: 50, allow_blank: true }
   validates :organization, presence: true
+  validates :repo_url, url: { allow_blank: true }, length: { maximum: 100, allow_blank: true }
```

Comment:
A limit of 100 seems potentially short for a URL of arbitrary length. We have URLs that are ~80 characters in the SoftEng organization, so it's not a stretch to imagine people having longer URLs. What do you think?